layout/_partial/blog_info.ejs

```diff
@@ -6,7 +6,7 @@
 
     <!-- LOGO -->
     <div class="something-else-logo mdl-color--white mdl-color-text--grey-600">
-        <a href="<%= theme.url.logo %>">
+        <a href="<% if(theme.url.logo !=null) { %> <%= theme.url.logo %> <% } else { %> # <% } %>">
```

Comment:
I would have written:
```
<a href="<%= theme.url.logo || '#' %>">
```
which is prettier, covers every falsy value and uses strict equality